install.sh

```diff
@@ -574,6 +585,23 @@ do_install_tar() {
     fi
 }
 
+setup_fapolicy_rules() {
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ] || [ -r /etc/rocky-release ]; then
+        verify_fapolicyd || return
+        # setting rke2 fapolicyd rules
+        cat <<-EOF >>"/etc/fapolicyd/rules.d/80-rke2.rules"
+allow perm=any all : dir=/var/lib/rancher/
+allow perm=any all : dir=/opt/cni/
+allow perm=any all : dir=/run/k3s/
+allow perm=any all : dir=/var/lib/kubelet/
+EOF
+        fagenrules --load || fatal "failed to load rke2 fapolicyd rules"
+        if [ -z "${INSTALL_RKE2_SKIP_RELOAD}" ]; then
+            systemctl restart fapolicyd
```

Comment:
I think this should go in there too... if this script is being run during system image creation (which is what that flag was added to support) then we don't need to load the rules either, as fapolicyd won't be running yet, for the same reason that systemd isn't running either.

```suggestion
        if [ -z "${INSTALL_RKE2_SKIP_RELOAD}" ]; then
            fagenrules --load || fatal "failed to load rke2 fapolicyd rules"
            systemctl restart fapolicyd
```